syntax = "proto3";

package csm.consensus;

enum RaftMessageType
{
    RequestVote = 0;
    RequestVoteReply = 1;
}

message RawRaftMessage
{
    RaftMessageType messageType = 1;
    bytes payload = 2;
}